### PR TITLE
keep and tag complete

### PR DESCRIPTION
**Pull request by joonhocho**

### CODE REVIEW

**Comment by joonhocho** [comment]
@2is10 
It is tested and working fine.

File: packrat/scripts/slider2.js [hunk 1]
```diff
@@ -219,12 +219,16 @@ var slider2 = slider2 || function () {  // idempotent for Chrome
...
       }).on("click", ".kifi-slider2-kept-lock", function (e) {
         if (e.target === this) toggleKeep($(this).closest(".kifi-slider2-keep-card").hasClass("kifi-public") ? "private" : "public");
-      }).on("click", ".kifi-slider2-kept-tag", function(e) {
+      }).on("click", ".kifi-slider2-keep-tag, .kifi-slider2-kept-tag", function(e) {
         if (e.originalEvent.tagboxClosed) {
           log('[tagbox:closed] ignore click event')();
           return;
         }
 
+        if ($(e.target).closest('.kifi-slider2-keep-tag').length) {
```
Comment (2is10) [inline packrat/scripts/slider2.js]: smaller/faster/clearer:

```
if (this.classList.contains('kifi-slider2-keep-tag')) {
```

or `$(this).hasClass`

**Comment by 2is10** [comment]
Looks great!

**Comment by joonhocho** [comment]
Changed from closest -> contains as suggested.

**Pull request merged**